acceptance/testconfig/all.json

```diff
@@ -0,0 +1,7 @@
+[
```

Comment:
Could this just be code in the acceptance test? Seems like the normal table-testing idioms should apply.

Reply:
There is something that we are trying to achieve in CI that makes it preferable to not have the full configuration statically set in the `acceptance_test`.

After talking to @ekcasey, we are leaning towards having a single environment variable that combines the paths and combinations to make it clearer.

Something like this:

```json
[
  {"pack": "", "pack_create_builder": "", "lifecycle": ""},
  {"pack": "", "pack_create_builder": "./pack-0.3.0", "lifecycle": "./lc-0.4.0"},
  {"pack": "./pack-0.3.0", "pack_create_builder": "", "lifecycle": "./lc-0.4.0"},
  {"pack": "./pack-0.4.0", "pack_create_builder": "./pack-0.3.0", "lifecycle": "./lc-0.4.0"}
]
```

We are also exploring passing a path to the suite config file instead of json through an environment variable.

Thoughts?

Reply:
Let's punt this to a future PR.